core/storage/suggestion/gae_models.py

```diff
@@ -320,6 +320,49 @@ def get_translation_suggestions_in_review_with_exp_id(
             .fetch(feconf.DEFAULT_QUERY_LIMIT)
         )
 
+    @classmethod
+    def get_multiple_translation_suggestions_in_review(
+            cls, thread_ids: List[str]
+    ) -> List['GeneralSuggestionModel']:
+        """Returns translation suggestions which are in review from multiple
+        thread ids.
+
+        Args:
+            thread_ids: list(str). Suggestion ids of translations that are in
+                review.
+
+        Returns:
+            list(SuggestionModel). A list of translation suggestions in review
+            with given suggestion ids.
+        """
+        return cast(
+            List[GeneralSuggestionModel],
+            GeneralSuggestionModel.get_multi(thread_ids)
+        )
+
+    @classmethod
+    def get_translation_suggestions_in_review_ids_with_exp_id(
+            cls, target_ids: List[str]
```

Comment:
Perhaps call it target_exploration_ids or target_exp_ids? Just "target" is a bit vague.

Reply:
Done